.changes/next-release/bugfix-AWSSDKforJavav2-86ff2d3.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "AWS SDK for Java v2",
+    "contributor": "",
+    "description": "Removed mapping for RESOLVED_CHECKSUM_SPECS execution attribute that was causing intermittent failures for S3 operations with checksums since 2.21.0 release."
```

Comment:
Can you include the specific exception details, so that people know whether this fix applies to the error they are seeing?